integration-tests/common/test_common.go

```diff
@@ -98,6 +105,20 @@ func (t *Test) DeployCluster(nodes int) {
 
 // DeployEnv Deploys the environment
 func (t *Test) DeployEnv(nodes int) {
+	imgParams := map[string]interface{}{
+		"image":   "795953128386.dkr.ecr.us-west-2.amazonaws.com/chainlink",
+		"version": "custom.2205e48ec7979b34fbc7a15ec2234bd16ca35122",
+	}
+
+	// override image parameters if necessary
+	if useLocalImg {
+		imgParams = map[string]interface{}{
+			"image":   "k3d-registry.local:12345/chainlink",
+			"version": "local",
+		}
+
+	}
```

Comment:
Ideally we can just pass in both the image and version as flags to our test run like so: `ginkgo -r --focus @ocr integration-tests/smoke -- --chainlink-image k3d-registry.local:12345/chainlink --chainlink-version local`.

We could also have presets like so: `ginkgo -r --focus @ocr integration-tests/smoke -- --chainlink [default/default-local/latest]`

Reply:
This is not a request, just thinking out loud :) Great job!